### DISCUSSION TITLE
Huge performance drop

### DISCUSSION THREAD

**Discussion by MrMondragon** [d8ahazard/sd_dreambooth_extension#827] [Q&A]
Hi there. I have been training models all week with crazy performances, like 500it/s on a dataset of 100+ images, running training sessions of 2000 epochs and 600K steps in less than 30 minutes.

But I screwed things up earlier today and had to perform a clean install this afternoon... and now I'm struglling to reach 6it/s.

Was able to set by batch size to values close to 200 and now I can't make it to 10. Any ideas on what could have happened?

BTW, I'm using a 3090 gpu.

Oh, and one more thing: before the re-install, these tests were running at about 13GB of VRAM. Now, with one tenth of the batch size and 100x less performance, I run at 23GB the hole time

Thanks in advance for any tips.

**Comment by cerega66** [comment]
500 it / s is some kind of space speed, even for the A100 it's too fast. 6 it/s looks much more realistic. For example, 400 it/s is achieved by using several A100s together. And for one - it's 80 it/s and then using voltaML. Most likely you had some kind of display problem, or something else. Although I can't speak for sure since I have a 3060 and max speed is 2 it/s.

**Reply by cerega66** [reply]
But I can give you tips for speeding up:
Use bf16
Use xformers
Try using 8 bit adam 

It would also be nice if you wrote down the full training parameters, maybe you only trained Lora.

**Comment by 1025307336** [comment]
What you installed later is the integration package?
